pkgs/applications/networking/instant-messengers/rambox/default.nix

```diff
@@ -21,8 +21,11 @@ in mkRambox rec {
     description = "Free and Open Source messaging and emailing app that combines common web applications into one";
     homepage = "https://rambox.pro";
     license = licenses.mit;
-    maintainers = with maintainers; [ ma27 ];
+    maintainers = with maintainers; [ ];
     platforms = ["i686-linux" "x86_64-linux"];
     hydraPlatforms = [];
+    knownVulnerabilities = [
+      "Electron 7.2.4 is EOL and contains at least the following vulnerabilities: CVE-2020-6458, CVE-2020-6460 and more (https://www.electronjs.org/releases/stable?version=7)"
```

Comment:
```suggestion
      "Electron 7.2.4 is EOL and contains at least the following vulnerabilities: CVE-2020-6458, CVE-2020-6460 and more (https://www.electronjs.org/releases/stable?version=7). Consider switching to ferdi"
```

Reply:
Done as it's the best alternative I'm aware of. Though I'm a little tempted to state that messenger aggregation is an unsolved CS problem after having used ferdi for a while %)

Reply:
I guess Android kind of solves it, in that if you want to read lots of messages you can setup all the clients as apps on Android phone(s) and sync the notifications to your desktop with KDE Connect. And then you'd set up screen-sharing for replying. But buying a phone is probably not anyone's first choice of solution.